### PR TITLE
Addbasic elements

### PR DESCRIPTION
**Pull request by Dygerydoo**
The content on this PR is finished, still need the addition of more elements, but meanwhile you can use the buttons and colors that already exists on this PR.

styleguide 20150421

### CODE REVIEW

File: app/assets/stylesheets/psique/components/_button.scss [hunk 1]
```diff
@@ -64,12 +49,112 @@
...
+    vertical-align: text-top;
+  }
+  span{
+    padding-left:3px;
+  }
+  &:hover{
+    @include linear-gradient(lighten($facebook,10), darken($facebook, 8));
+  }
+}
+.btn--gplus{
```
Comment (d2bit) [inline app/assets/stylesheets/psique/components/_button.scss]: SUG: these could be extracted to a mixin
```sass
@mixin btnSocial($gradientColor, $borderColor) {
  @include linear-gradient(lighten($gradientColor, 8), $gradientColor);
  .fa::before{
    padding: 5px 12px 5px 0px;
    border-right: 1px solid $borderColor;
    position: relative;
    left: -12px;
    top: 1px;
    vertical-align: text-top;
  }
  span{
    padding-left:3px;
  }
  &:hover{
    @include linear-gradient(lighten($gradientColor,10), darken($gradientColor, 8));
  }
}

.btn--facebook { @include btnSocial($facebook, $navyblue); }
.btn--gplus { @include btnSocial($gplus, $grana); }
.btn--twitter { @include btnSocial($twitter, $navyblue); }
```

**Comment by maxvlc** [comment]
:+1: 
I prefer this kind of view the elements. In this way is really easy to view how use it.

File: app/views/psique/documentation/elements.html.haml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+      %span.sc .btn--s  
+  .col-3
+    .btn--a.btn--xs Extra Small Button
+    %p.col-box.dark-grey.codehilite 
+      When your div has the class 
+      %span.nc .btn--a/b/c
+      append to this
+      %span.sc .btn--xs
+
+.grid.main
```
Comment (d2bit) [inline app/views/psique/documentation/elements.html.haml]: Don't track work in progress. A PR should show only finished work. :stuck_out_tongue:

File: app/assets/stylesheets/psique/components/_button.scss [hunk 3]
```diff
@@ -64,12 +49,112 @@
...
+  }
+}
+
+// Social
+
+.btn--facebook{
+  @include linear-gradient(lighten($facebook, 8), $facebook);
+  .fa::before{
+    padding: 5px 12px 5px 0px;
+    border-right: 1px solid $navyblue;
```
Comment (Fernan2) [inline app/assets/stylesheets/psique/components/_button.scss]: [SUG] Designers like names like `navyblue`, but developers maybe understand better something like `azul-facebook`

File: app/assets/stylesheets/psique/components/_button.scss (hunk 1 above)
Comment (Fernan2) [inline app/assets/stylesheets/psique/components/_button.scss]: Nice DRYing!

File: app/assets/stylesheets/psique/foundation/_config.scss [hunk 4]
```diff
@@ -9,25 +9,68 @@
 //   Colors
 // -------------------------------------
 
-// ----- Palette ----- //
+// ----- Palette Vars ----- //
+
+// Main Colors
```
Comment (Fernan2) [inline app/assets/stylesheets/psique/foundation/_config.scss]: Although I'm not a comments lover, this seem a good place to add a link (commented, of course) pointing to where we can see the colors palette

**Comment by Fernan2** [comment]
We already need the FB button for the login page, so merging this as is (even if it's not finished) could be a good idea (maybe after DRYing social buttons)

File: app/assets/stylesheets/psique/components/_button.scss (hunk 3 above)
Comment (Dygerydoo) [inline app/assets/stylesheets/psique/components/_button.scss]: Facebook color has name $facebook (maybe need blue word {I mean like $blue-facebook}) $navyblue is a primary color, "azul marino" xD, but I try to approach the names to code side.

File: app/assets/stylesheets/psique/components/_button.scss (hunk 1 above)
Comment (Dygerydoo) [inline app/assets/stylesheets/psique/components/_button.scss]: "Oido cocina" :+1:

**Comment by d2bit** [comment]
:+1:

**Comment by scruti** [comment]
:+1:

**Pull request merged**